students/JasneetChandok/Lesson05/mailroom3.py

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env python3.6
+
+'''Lesson 5 - Mailroom Part 3'''
+
+# Importing Systems Module
+try:
+    import sys
+except ImportError:
+    print("The sys module could nout be imported!")
+
+
+# Starter list of existing donors
+donors = {'jasneet paramjit': [50, 200, 500],
+          'simran kaur': [100, 200, 500],
+          'manikaran chandok': [100, 200, 500, 680],
+          'manveen chandok': [10, 200, 65],
+          'sanjam patwalia': [200, 50]}
+
+
+# List of existing donor
+def existingdonors_list():
+    donor_list = "\n ".join(donor.title()for donor in donors)
+    print("\nList of Existing Donors:", "\n", donor_list, "\n")
+
+
+# Add donation to an existing donor
+def add_donation():
+    try:
+        donor_selection = input("Enter 'Full Name' of the donor >")
+        donor_donation = input("\nEnter the 'Amount' donated >")
+        donor = donor_selection.strip().lower()
+        if donor in donors:
+            donor_amount = donors.get(donor)
+            donor_amount.append(int(donor_donation))
+            print(f'\nDonation amount of "{donor_donation}" has been recorded for {donor.title()}!!\n')
+            print('''Thank you note to share out with the donor below:\n
+              Dear {}s
+
+              Thank you for your very kind donation of ${:.2f}.
+              It will be put to very good use.
+
+                             Sincerely,
+                                -The Team
+                '''.format(donor.title(), int(donor_donation)))
+        else:
+            print('\nDonor name entered does not exist in the list!!')
+    except (EOFError, KeyboardInterrupt):
+        print("""\nA user induced error occured. Please check keys pressed!""")
+
+# Create a record for a new donor
+def new_donor():
+    try:
+        donor_name = input("Enter 'Full Name' of the donor >")
+        new_donation = int(input("\nEnter the 'Amount' donated >"))

```

Comment:
Same goes for here, you can run into  an error if you input a string instead of int